src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,116 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+/**
+ * Main driver class for Duke project.
+ * Chatbot that manages a task list.
+ */
+public class Duke {
+    public static void main(String[] args) {
+        boolean isRunning = true;
+        TaskStorage taskStorage = new TaskStorage();
+        TaskList taskList;
+
+        // Load tasks from duke.TaskStorage
+        try {
+            taskList = new TaskList(taskStorage);
+        } catch (DukeException e) {
+            System.out.printf("[!] %s\n", e.getMessage());
+            return;
+        }
+
+        Ui ui = new Ui("Pong");
+        ui.init();
+
+        while (isRunning) {
+            try {
+                Parser parser = ui.getParsedInput();
+                isRunning = Duke.handleInput(parser, taskList, ui);
+            } catch (DukeException e) {
+                ui.printException(e);
+            }
+        }
+
+        ui.exit();
+    }
+
+    private static boolean handleInput(Parser parser, TaskList taskList, Ui ui) throws DukeException {
+        Task task;
+        switch (parser.getCommand()) {
+        case "bye":
```

Comment:
perhaps it would be better to consider the case that the user might input the correct word but a different capitalisation? For eg. instead of bye, the user inputs Bye or BYE.

Reply:
Definitely, will work on that.